### ISSUE TITLE
IDE0053 (Use expression body for lambda expressions) is being reported as a warning by default in latest 17.4 dogfooding builds

### ISSUE THREAD

**Issue by mavasani** [dotnet/roslyn#63160]
Create a new C# console app with following code:

```csharp

var x = new Action(() =>
{
    Console.WriteLine("Hello World!");
});
```

Notice that IDE0053 is being reported as a warning instead of hidden diagnostic:

![image](https://user-images.githubusercontent.com/10605811/182587604-8dc84c37-f978-422e-9fda-2a8161d9d324.png)

**Comment by mavasani** [comment]
Tagging @Youssef1313. I am wondering if this is due to https://github.com/dotnet/roslyn/pull/62675 or other recent IDE code style analyzer refactoring PRs.

**Comment by CyrusNajmabadi** [comment]
Clsoing out due to lacck of broad feedback.
